### PR TITLE
cleanup `.later`

### PR DESCRIPTION
**Pull request by bekzod**

### CODE REVIEW

**Comment by bekzod** [comment]
@rwjblue would be nice to get this merged :P

File: lib/index.ts [hunk 1]
```diff
@@ -393,29 +393,12 @@ export default class Backburner {
...
     let target;
     let methodOrTarget;
-    let methodOrWait;
     let methodOrArgs;
 
     if (length === 0) {
       return;
     } else if (length === 1) {
       method = args.shift();
-    } else if (length === 2) {
```
Comment (stefanpenner) [inline lib/index.ts]: is this a safe change? Seems like it is breaking no?

Comment (stefanpenner) [reply lib/index.ts]: I would love to make this change, I really dislike the complexity it aims to remove.

Comment (bekzod) [reply lib/index.ts]: nope it is not breaking, if you look `last else block` already handles `2+` arguments length and there is test coverage already for it when `bb.later(()=>{}, 123)`

Comment (stefanpenner) [reply lib/index.ts]: lets shipit then

**Comment by stefanpenner** [comment]
needs another rebase POST the TS change we just merged.

**Comment by bekzod** [comment]
done

**Comment by bekzod** [comment]
rebased

**Pull request merged**